content/docs/tasks/telemetry/kiali/index.md

```diff
@@ -0,0 +1,400 @@
+---
+title: Visualizing Your Mesh
+description: This task shows you how to visualize your services within an Istio mesh.
+weight: 49
+keywords: [telemetry,visualization]
+---
+
+This task shows you how to visualize different aspects of your Istio mesh.
+
+As part of this task, you install the [Kiali](https://www.kiali.io) add-on
+and use the web-based graphical user interface to view service graphs of
+the mesh and your Istio configuration objects. Lastly, you use the Kiali
+Public API to generate graph data in the form of consumable JSON.
+
+The [Bookinfo](/docs/examples/bookinfo/) sample application is used as the
+example application throughout this task.
+
+## Before you begin
+
+Create a secret in your Istio namespace with credentials that you use to
+authenticate to Kiali. See the
+[Helm README](https://github.com/istio/istio/blob/master/install/kubernetes/helm/istio/README.md#installing-the-chart)
+for details, but here is an example command you can modify and run to create a secret:
+
+```bash
+USERNAME=$(echo -n 'admin' | base64)
+PASSPHRASE=$(echo -n 'mysecret' | base64)
+NAMESPACE=istio-system
+kubectl create namespace $NAMESPACE
+cat <<EOF | kubectl apply -f -
+apiVersion: v1
+kind: Secret
+metadata:
+  name: kiali
+  namespace: $NAMESPACE
+  labels:
+    app: kiali
+type: Opaque
+data:
+  username: $USERNAME
+  passphrase: $PASSPHRASE
+EOF
+```
+
+Once the Kiali secret is created, install Kiali via Helm by following
+[these instructions](/docs/setup/kubernetes/helm-install/). Note that you need
+to use the `--set kiali.enabled=true` option when running the `helm` command.
+
+For example:
+
+{{< text bash >}}
+$ helm template --set kiali.enabled=true install/kubernetes/helm/istio --name istio --namespace istio-system > $HOME/istio.yaml
+$ kubectl apply -f $HOME/istio.yaml
+{{< /text >}}
+
+> {{< idea_icon >}} This Task does not discuss Jaeger and Grafana; however, if
+you have them already installed in your cluster and you want to see how Kiali
+integrates with both of them, you need to pass additional arguments to the
+`helm` command. For example:
+
+{{< text bash >}}
```

Comment:
I tried that, it doesn't render any differently. I did get the code block to become part of the note, but only by deleting the blank line between the note text and the text bash - however, when I do that, the linter gives me an error message:

```
content/docs/tasks/telemetry/kiali/index.md:57: MD031 Fenced code blocks should be surrounded by blank lines
```

So to avoid having the "make lint" job block this PR I put the blank line back.

Indenting the code block (I tried indenting the text bash markers as well as not indenting the text bash markers) does not help. It doesn't render any differently. However, when I commit my next round of changes, I will indent the code block with four spaces.